surveys/sphinx-survey-2025.yml

```
@@ -14,81 +14,70 @@
 
-title: "Sphinx Survey 2025"
+title: "Sphinx Documentation Survey (2025)"
 questions:
-- title: "Sphinx Usage Survey 2025"
+- title: "Sphinx Documentation Survey (2025)"
   type: statement
   description: >-
-    Sphinx Documentation User Survey 2025
+    We’re running this survey to understand requirements for documentation
+    tools in 2025, how they are used, and how well tools support their users.
+    This survey is being run on behalf of the [Sphinx] community, but is
+    open to anyone who works with or contributes to documentation, regardless
+    of whether you use Sphinx.
 
-    Help the Sphinx community understand who is using Sphinx, how they are
-    using it, and how well it supports the people using it. We will use this
-    information to aid community discussion about improvements to Sphinx.
+    We will use this information to aid community discussion about improvements
+    to Sphinx. Results will be public, with any identifying information removed.
+    Open text field questions may be quoted, but will not be shown in full.
 
-    Results will be public with identifying information removed. Open text
-    field questions may be quoted, but will not be shown in full.
+    This survey is run by the Sphinx community and [Quansight Labs]; survey
+    data will be deleted from any private accounts once made public. If you
+    have any questions, contact Tania Allard (Director of Quansight Labs)
+    at tallard@quansight.com and mention Sphinx Survey 2025 in the subject line.
 
-    This survey is run by the Sphinx community and Quansight Labs; survey 
-    data will be deleted from any private accounts once made public. If you 
-    have any questions, reach out to tallard@quansight.com and mention 
-    Sphinx Survey 2025 in the subject line.
+    [Sphinx]: https://www.sphinx-doc.org/
+    [Quansight Labs]: https://labs.quansight.org/
 
-- title: "Tell us about you"
-  description: >-
-    First, we'd like to know about how you work and what tools you use.
-  type: statement
-
-# Motivation: Understand how the responder views themself in relation to Sphinx.
+# Motivation: Understand how the responder views themselves in relation to Sphinx.
 - title: "What is your relation to the Sphinx project?"
   multiple: true
   choices:
   - "I read documentation that is built with Sphinx"
   - "I use Sphinx to build documentation"
-  - "I build extensions for Sphinx"
-  - "I maintain a Sphinx theme"
-  - "I contribute to Sphinx"
+  - "I maintain or have contributed to a Sphinx extension"
+  - "I maintain or have contributed to a Sphinx theme"
+  - "I have contributed to the Sphinx project"
   - "None of the above"
 
 # Motivation: Understand the responder's self-reported background and skills.
-- title: "Which of the following roles is closest to your job?"
+- title: "Which of the following best describes your occupation?"
   other: true
   choices:
   - "Academic researcher"
   - "Data scientist"
   - "Developer / Programmer"
```

```suggestion
  - "Software Developer / Programmer / Engineer"
```

`Software engineer` should be listed somewhere explicitly

On https://pigweed.dev I am the only technical writer and the 25-50 other people that contribute to the docs all have the literal job title of `software engineer`

---

(Closing the loop) I don't think my original comment is too important. This is a pretty straightforward question and there is low risk of ambiguity causing noisy responses

---

I have no strong feelings here so leaving it to y'all